addons/website_slides/models/slide_slide.py

```diff
@@ -533,6 +534,16 @@ def _compute_website_url(self):
                 base_url = slide.channel_id.get_base_url()
                 slide.website_url = '%s/slides/slide/%s' % (base_url, slug(slide))
 
+    @api.depends('channel_id.website_id.domain', 'website_url')
+    def _compute_website_share_url(self):
+        for slide in self:
```

Comment:
To skip the if you may do
```
def _compute_website_share_url(self):
    self.website_share_url = False
    for slide in self.filtered('id'):
        if slide.is_preview:
        ...
```

Reply:
I have removed that part. The goal was to share a SEO-friendly URL while sharing with social media for the public URL as explained in the task comment.
But you're right if we set after sharing a URL the preview flag to false, the shared URL will lead to a 403. Now, it is no longer the case.

Reply:
Would still add the first line ``self.website_share_url = False`` . That way we force init the field to False and avoid any cache miss (even if now default value works in place of a cache miss I think).

Reply:
Oops, yes I missed that (Thought it was about the removal of _compute_website_share_url in channel). It is done, will push soon